src/Control/Carrier/Cull/Church.hs

```diff
@@ -50,7 +51,7 @@ runCullM :: (Applicative m, Monoid b) => (a -> b) -> CullC m a -> m b
 runCullM leaf = runCull (liftA2 mappend) (pure . leaf) (pure mempty)
 
 -- | @since 1.0.0.0
-newtype CullC m a = CullC (ReaderC Bool (NonDetC m) a)
+newtype CullC m a = CullC { runCullC :: ReaderC Bool (NonDetC m) a }
```

Comment:
I introduced selectors for a few types like this, but I don’t export them. I could maybe have used `coerce . hom` in the algebra instead of `runCullC . hom`, but it should be eliminated either way, and this way is clearer.